src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -28,6 +24,13 @@ import {
   getDomainMetadataMessageDescriptorsIds,
   getColonyRoleSetMessageDescriptorsIds,
 } from '~utils/colonyActions';
+import { useDataFetcher } from '~utils/hooks';
+import { ipfsDataFetcher } from '../../../../core/fetchers';
+
+import { ColonyAndExtensionsEvents } from '~types/index';
```

Comment:
I think we have here bad ordering of imports 😆

Reply:
Good catch